ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml

```diff
@@ -20,4 +20,79 @@ LGRP_DC:
   is_canonical: true
   implements:
     - LGRP
-    - DC
\ No newline at end of file
+    - DC
+
+LGRP_OS_SCZ:
```

Comment:
@biotvandenberg did you change anything here? it looks identical to what existed before.

Reply:
LGRP_OS_SCZ didn't exist before